microsoft/testsuites/core/provisioning.py

```diff
@@ -251,9 +253,11 @@ def _smoke_test(
                 if not is_ready:
                     serial_console = node.features[SerialConsole]
                     serial_console.check_panic(saved_path=log_path, stage="reboot")
-                    raise LisaException(
-                        f"Cannot connect to [{node.public_address}:{node.public_port}],"
-                        f" error code: {tcp_error_code}, no panic found in serial log"
+                    raise TcpConnetionException(
```

Comment:
This message should be different with previous one in Ln 228. They should specify it happens on boot or reboot.

Reply:
updated